telegram/ext/callbackqueryhandler.py

```diff
@@ -34,10 +34,15 @@ class CallbackQueryHandler(Handler):
         pass_update_queue (optional[bool]): If the handler should be passed the
             update queue as a keyword argument called ``update_queue``. It can
             be used to insert updates. Default is ``False``
+        pass_job_queue (optional[bool]): If the callback should be passed the job queue as a
+            keyword argument called ``job_queue``. It can be used to schedule new jobs.
+            Default is ``False``
     """
```

Comment:
`callback` is the first argument to `__init__`, also you added a `to` there. It's actually the other way: `job_queue´ is passed to the `callback` function.